Client/src/i18n/ru/index.js

```diff
@@ -2,9 +2,8 @@ import admin from './admin';
 import components from './components';
 import errors from './errors.js';
 
-
 export default {
   ...components,
   Admin: {...admin},
   Errors: {...errors},
-}
+}
```

Comment:
Please, add the new line to the end of the files.
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file